### DISCUSSION TITLE
Feature Request : youtube tablet community and age restriction bypass

### DISCUSSION THREAD

**Discussion by plasma3000** [ReVanced/revanced-discussions#759] [Question]
1. Youtube community is not working on android tablet. it says "Community posts aren't currently available on this device. You can access Community tab content from your desktop or other supported devices." (idk if this is the same on ipad)
2. Youtube often says "age-restricted" even its not violence or sexual related.  I saw this browser extention,(https://microsoftedge.microsoft.com/addons/detail/age-restriction-bypass-fo/nbonnacihdagjflaacjbjcdbpfoomgnp) wonder if revanced dev can add this function.

thanks, sorry if my english was bad.

**Comment by oSumAtrIX** [comment]
https://github.com/ReVanced/revanced-discussions/discussions/118
